hypertrace-trace-enricher/trace-reader/src/main/java/org/hypertrace/trace/reader/entities/AttributeValueConverter.java

```diff
@@ -28,6 +29,22 @@
     }
   }
 
+  public static Optional<String> convertToString(LiteralValue literalValue) {
```

Comment:
This probably belongs in the coercer utils - it's already public, works in primitives (/strings) and literals, and also that's what this is doing, coercing (albeit in the opposite direction of the existing definitions)